Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Start with empty ubuntu machine
-FROM ubuntu:15.04
+FROM ubuntu:xenial
```

Comment:
there will be issues when obtaining output from autograder if changed to 18.04 based on https://github.com/autolab/Tango/issues/153

there are relevant patch in there to address it. Has that been added?

Reply:
xenial is actually 16.04, but I just changed it to 18.04. This Dockerfile is for the Tango Dockerfile, I did not modify the autograding Dockerfile in this PR which is still on 14.04 (should fix that soon though)